### PR TITLE
Persist sidebar state once a selection is made

### PR DESCRIPTION
**Pull request by arunshenoy99**

### CODE REVIEW

File: src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const GetStartedExperience = () => {
 	const [ isLoaded, setisLoaded ] = useState( false );
 	const [ wpComfortLevel, setWpComfortLevel ] = useState( '0' );
 
-	const { setCurrentOnboardingData } = useDispatch( nfdOnboardingStore );
+	const { setCurrentOnboardingData, setSidebarActiveView } = useDispatch( nfdOnboardingStore );
```
Comment (officiallygod) [inline src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/index.js:24]: Can't we combine it with the useDispatch on line 33?

File: src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/PrimarySite/index.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ import { translations } from '../../../../../utils/locales/translations';
 
 const StepPrimarySetup = () => {
 
-	const { setDrawerActiveView, setIsSidebarOpened, setIsDrawerSuppressed, setIsHeaderNavigationEnabled } = useDispatch(
+	const { setDrawerActiveView, setIsDrawerSuppressed, setIsHeaderNavigationEnabled, setSidebarActiveView } = useDispatch(
```
Comment (officiallygod) [inline src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/PrimarySite/index.js:17]: Line 40 has another useDispatch, I knw this existed earlier but can you put it here?

File: src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/SecondarySite/index.js [hunk 3]
```diff
@@ -13,12 +13,12 @@ import { translations } from '../../../../../utils/locales/translations';
 
 
 const StepPrimarySetup = () => {
-	const { setDrawerActiveView, setIsSidebarOpened, setIsDrawerSuppressed, setIsHeaderNavigationEnabled } = useDispatch(
+	const { setDrawerActiveView, setSidebarActiveView, setIsDrawerSuppressed, setIsHeaderNavigationEnabled } = useDispatch(
```
Comment (officiallygod) [inline src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/SecondarySite/index.js:16]: Same as PrimarySite line 31 has a useDispatch.

File: src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/index.js (hunk 1 above)
Comment (arunshenoy99) [reply src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/index.js]: Hmm, looks like these double useDispatches had already made it to trunk

**Review by officiallygod — APPROVED** [review]

**Pull request merged**